plugins/env/test/models/environment_variable_test.rb

```diff
@@ -43,6 +43,65 @@
       EnvironmentVariable.env(Project.new, 123).must_equal({})
     end
 
+    describe "get env vars from GitHub" do
+      with_env DEPLOYMENT_ENV_REPO: "organization/repo_name"
+      before do
+        project.use_env_repo = true
+      end
+
+      it "returns a processed env hash" do
+        stub_github_api(
+          "repos/organization/repo_name/contents/generated/foo/pod100.env",
+          "FROM_REPO_VAR_ONE=one\nVAR_TWO=two\n"
+        )
+        expected_result = {"FROM_REPO_VAR_ONE" => "one", "VAR_TWO" => "two"}
+        EnvironmentVariable.env(project, deploy_group).must_equal expected_result
+      end
+
+      it "returns a merges db env and repo env hashes" do
```

Comment:
keep it low-tech / user-story style
```suggestion
      it "merges repo env into db env" do
```